docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx

```diff
@@ -17,10 +17,10 @@ import { Card, Dropdown, Grid, Header, SubNavigation, TitleBar } from 'react-cm-
 import { Link } from 'react-router';
```

Comment:
Are we planning on renaming `<Dropdown>` => `<DropdownDeprecated>`?

Reply:
I don't think we should rename these in `./src`, but in `./docs/src` my thinking was/is to better identify what is deprecated. I could be wrong. Let me know.